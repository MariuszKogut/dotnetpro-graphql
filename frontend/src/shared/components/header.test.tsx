import React from 'react'
import { render } from '@testing-library/react'
import Header from './header'

jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    push: jest.fn(),
  }),
}))

describe('Header', () => {
  it('renders without errors', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot()
  })
})
