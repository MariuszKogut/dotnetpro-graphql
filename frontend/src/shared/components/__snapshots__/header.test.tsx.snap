// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without errors 1`] = `
<div>
  <header>
    <nav
      class="navbar navbar-expand-md navbar-dark bg-dark fixed-top"
    >
      <span
        class="navbar-brand"
      >
        Customer App
      </span>
      <button
        aria-controls="responsive-navbar-nav"
        aria-label="Toggle navigation"
        class="navbar-toggler collapsed"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="navbar-collapse collapse"
        id="responsive-navbar-nav"
      >
        <div
          class="mr-auto navbar-nav"
        >
          <div
            class="dropdown nav-item"
          >
            <a
              aria-expanded="false"
              aria-haspopup="true"
              class="dropdown-toggle nav-link"
              href="#"
              role="button"
            >
              Kunden
            </a>
          </div>
        </div>
        <div
          class="navbar-nav"
        />
      </div>
    </nav>
  </header>
</div>
`;
